docs/design/queue-level-scheduling-policy.md

```diff
@@ -0,0 +1,609 @@
+In the current design, Volcano adopts a global scheduling policy defined in a centralized configuration file. 
+As a result, all queues and jobs within the cluster share the same set of scheduling actions and plugins, 
+regardless of their specific characteristics. 
+However, with the increasing adoption of diverse workloads such as 
+training and inference tasks even-within the same tenant—there is a growing demand for differentiated scheduling strategies. 
+To address this, it is essential to introduce support for scheduling policies at both the queue and job levels.
+
+# 1、Changes to the CRD
+
+## 1.1、Extend the queue CRD
+
+Add the `SchedulerPolicy` in the `QueueSpec`, allowing the queue to define its own scheduling policy configuration.
+
+```go
+type QueueSpec struct {
+    Weight     int32
+    Capability v1.ResourceList
+    State QueueState
+    Reclaimable *bool
+    ExtendClusters []Cluster
+    Guarantee Guarantee `json:"guarantee,omitempty" protobuf:"bytes,4,opt,name=guarantee"`
+    Affinity *Affinity `json:"affinity,omitempty" protobuf:"bytes,6,opt,name=affinity"`
+    Type string `json:"type,omitempty" protobuf:"bytes,7,opt,name=type"`
+    Parent string `json:"parent,omitempty" protobuf:"bytes,8,opt,name=parent"`
+    Deserved v1.ResourceList `json:"deserved,omitempty" protobuf:"bytes,9,opt,name=deserved"`
+    Priority int32 `json:"priority,omitempty" protobuf:"bytes,10,opt,name=priority"`
+	
+    // Add SchedulerPolicy to point to the scheduling policy defined in the configuration file
+    SchedulerPolicy string
+}
+
+```
+
+## 1.2、Extend the Job annotation
+
+To avoid potential issues caused by modifying the `JobSpec` directly, 
+job-level scheduling policies are specified via `annotations` in the Job definition.
```

Comment:
`annotations` is not a official practice.

Reply:
Yeah actually we recommend to use it at queue level, there are two aspects consideration:
1. Queue level is more common in multi tenants scenario, which is recommended.
2. If new fields are added to the pod and job APIs, a lot of adaptation is required because of this major API change, while setting policies on queues will be less expensive for users.